public/app/routes/routes.tsx

```diff
@@ -514,12 +514,16 @@ export function getAppRoutes(): RouteDescriptor[] {
       ),
     },
     config.featureToggles.exploreMetrics && {
+      ...(config.featureToggles.exploreMetricsUseExternalAppPlugin
+        ? getRouteForAppPlugin('grafana-exploremetrics-app')
```

Comment:
I believe that the `roles` config should be taken care of by the plugin's `action` type:

https://github.com/grafana/explore-metrics/blob/main/src/plugin.json#L35

but I'm not sure if we need to specify `chromeless: false` here.